components/markdown/src/markdown.rs

```diff
@@ -326,7 +329,13 @@ pub fn markdown_to_html(
         for (event, mut range) in Parser::new_ext(content, opts).into_offset_iter() {
             match event {
                 Event::Text(text) => {
-                    if let Some(ref mut _code_block) = code_block {
+                    if lazy_async_image && is_image_alt {
```

Comment:
Can we not just push back the Text event in that case and let pulldown-cmark handle it?

Reply:
Indeed, this addition is redundant. I have removed it.